src/app/routes/homePage/getInitialData/page-config.ts

```diff
@@ -1,10 +1,200 @@
+import { Services } from '../../../models/types/global';
+
 type Envs = 'test' | 'live';
 
-const HOME_PAGE_CONFIG: { [key: string]: { [key in Envs]: string } } = {
+const HOME_PAGE_CONFIG: { [service in Services]: { [env in Envs]: string } } = {
   kyrgyz: {
     test: 'cm7682qz7v1t',
     live: 'crg7kj2e52nt',
   },
+  afaanoromoo: {
+    test: 'c93v2kkzl24t',
+    live: 'c44dyn08mejt',
+  },
+  afrique: {
+    test: 'cjln1ww62p1t',
+    live: 'cg58k91z9e3t',
+  },
+  amharic: {
+    test: 'cx35pee6zw8t',
+    live: 'cg58k91re1gt',
+  },
+  arabic: {
+    test: 'cx35pee6zvpt',
+    live: 'cq01ke64p2et',
+  },
+  archive: {
+    test: '',
+    live: '',
+  },
```

Comment:
One for another day: if we remove these, we will get typescript errors. 

We might have a couple of solutions:
- Remove them, and add the `@ts-ignore` annotation to the entire fire, so that it doesn't fail type checking
- Refactor of Services type to differentiate between World Service and others such as news, sport & nations, and import the World Service type here only... 🤔 

I'd maybe like to get a second opinion on what might be the best approach

Reply:
Sounds like a plan!